### DISCUSSION TITLE
Feature request: Allow filtering by asset name

### DISCUSSION THREAD

**Discussion by chrishiste** [dagster-io/dagster#24689] [Ideas]
It would super useful if we could filter by asset names.
![ScreenShot 2024-09-24 at 14 43 21@2x](https://github.com/user-attachments/assets/6d6178bd-391f-4933-8491-ac633a31bfaf)

**Comment by garethbrickman** [comment]
Could you show a full screenshot for context of where in the UI this is?

**Reply by chrishiste** [reply]
![ScreenShot 2024-09-25 at 13 43 52@2x](https://github.com/user-attachments/assets/2d711546-4ebd-42f1-b4d7-fd360e578bad)

**Comment by garethbrickman** [comment]
Created a feature request issue: https://github.com/dagster-io/dagster/issues/24744
